CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Features
 - Added support for Snowflake query tags at the connection and model level ([#1030](https://github.com/fishtown-analytics/dbt/issues/1030), [#2555](https://github.com/fishtown-analytics/dbt/pull/2555/))
 - Added option to specify profile when connecting to Redshift via IAM ([#2437](https://github.com/fishtown-analytics/dbt/issues/2437), [#2581](https://github.com/fishtown-analytics/dbt/pull/2581))
+- Add two new flags `--use-colors` and `--no-use-colors` to `dbt run` command to enable or disable log colorization from the command line [#2622](https://github.com/fishtown-analytics/dbt/pull/2622)
```

Comment:
Please also link to the issue, and add a link to this PR/your username to the "contributors" section below. That way we can make sure you get credit!